client/my-sites/checkout/checkout-thank-you/test/index.js

```diff
@@ -91,12 +90,13 @@ describe( 'CheckoutThankYou', () => {
 		} );
 
 		test( 'Show WordPressLogo when there are no purchase but a receipt is present', () => {
-			render(
+			const { container } = render(
 				<Provider store={ store }>
 					<CheckoutThankYou { ...defaultProps } receiptId={ 12 } />
 				</Provider>
 			);
-			expect( screen.getByTestId( 'wordpress-logo' ) ).toBeVisible();
+			// TODO: `WordPressLogo` should probably be updated to pass through props like `data-testid`
+			expect( container.querySelector( '.checkout-thank-you__logo' ) ).toBeVisible();
```

Comment:
An unrelated thought: Should the `<WordPressLogo />` component have a text alternative so that we could query it by name? e.g.

```suggestion
			expect( getByRole( 'img', { name: 'WordPress' } ) ).toBeVisible();
```

I guess it depends the context on how it's used, and that we might be using it presentationally, but other cases it stands out as inconsistent and strange that it's strictly not supported to assign a text alternative:

https://github.com/Automattic/wp-calypso/blob/193efb4c4ed498567a0077b75244cf7cf10eb599/client/a8c-for-agencies/sections/referrals/primary/referrals-overview/layout-body-content.tsx#L139-L141

Reply:
I recently looked into all the logo componentry (#99598) and it's quite a mess. Some even have `title` tags built in. Unfortunately it's deep in the backlog because we have other higher impact things to do.

I even object to the existence of this specific test, that queries a logo that seems 100% presentational 😅 That's not what jsdom tests are for, IMO.

Reply:
> I even object to the existence of this specific test, that queries a logo that seems 100% presentational 😅 That's not what jsdom tests are for, IMO.

Yeah, that's a good point. Feels like either it's presentational and it's not worth checking the existence in the tests, or we care enough about its presence in the page that it's not actually presentational.